docs/earth/xdggs.md

```diff
@@ -0,0 +1,59 @@
+---
+jupytext:
+  text_representation:
+    format_name: myst
+kernelspec:
+  display_name: Python 3
+  name: python
+---
+
+# xdggs: DGGSIndex
+
+````{grid}
+```{grid-item}
+:columns: 3
+```{image} https://xdggs.readthedocs.io/en/latest/_static/logos/xdggs_logo.png
+---
+alt: xdggs logo
+width: 200px
+align: center
+---
+```
+```{grid-item}
+:columns: 9
+```{seealso}
+Learn more at the [xdggs](https://xdggs.readthedocs.io/en/latest/) documentation page.
+```
+````
+
+## Highlights
+
+xdggs provides a unified API for interacting with various Discrete Global Grid Systems (DGGS). It does so by parsing out grid metadata from the spatial coordinate containing the cell ids and persisting it on a "meta-index", an index that wraps another index (most commonly a `pandas` index).
+
+## Example
+
+Here's a dataset on a [healpix](https://healpix.sourceforge.io/html/intro.htm) grid:
+
+```{code-cell} python
+import xdggs
+
+ds = xdggs.tutorial.open_dataset("air_temperature", "healpix")
+ds
+```
+
+with that, we can decode the metadata:
+
+```{code-cell} python
+decoded = ds.dggs.decode()
+decoded
+```
+
+Note how the `cell_ids` coordinate is now associated with a `HealpixIndex`? This makes it harder to lose the parsed information, and thus having to parse it again.
+
+We can also have a look at the parsed metadata:
+
+```{code-cell} python
+decoded.dggs.grid_info
+```
+
+In a future version of `xdggs` this may also support more operations, like alignment, selection using parent cell ids, or lazy coordinates for cell centers / cell boundaries.
```

Comment:
Interesting that you don't use it for selection yet.

I think for visual interest, it would be good to show `.explore()` too. Really show off the grid that way.

Reply:
we use the wrapped index (mostly pandas, although I'm working on a set-of-ranges based index) for the exact selection of cell ids, but Benoit is planning to add selection based on virtual coordinates (cell centers, cell boundaries), such that you could do `ds.sel(lat=..., lon=...)` or `ds.sel(cell_boundaries=geometry_array)` and the DGGS index would transform those to cell ids, then call the wrapped index to actually translate to index space.